Makefile

```diff
@@ -5,7 +5,7 @@ SHELL := /bin/bash
 run: make-apply
 
 make-apply:
-	cd $(PROJECT_DIR) && AUTO_APPROVE="-auto-approve" make
+	@export AUTO_APPROVE="-auto-approve" && cd $(PROJECT_DIR) && (if [[ "${backendApplicationHosting}" =~ "kubernetes" ]]; then make apply; else make apply-without-eks; fi)
```

Comment:
Are you going to implement a `teardown` associated with SAM?

Reply:
This is the makefile at the root of the module, the one in the templates dir is responsible for teardown. But yeah, we should probably have an equivalent `teradown-without-eks` make target that just doesn't run the teardown eks stuff.